{
  "bug_location": "Unknown",
  "severity": 3,
  "categories": [
    "Unknown"
  ],
  "root_cause": "N/A - Analysis failed",
  "issue_number": 7462,
  "title": "After a user starts TiKV, other users will not be able to start TiKV"
}